### ISSUE TITLE
Accessibility: Screen reader does not read dates while navigating using arrow keys

### ISSUE THREAD

**Issue by sarveshnam** [Hacker0x01/react-datepicker#1624]
**Expected behavior**
NVDA and Windows narrator should read out the dates on navigating through date picker using arrow keys. It should pronounce the date being selected (e.g. day-8, day-12 etc.). 

**Actual behavior**
NVDA and Windows narrator doesn't read out the dates on navigating through date picker using arrow keys. It just says "blank" while navigating through dates in date picker.,

**Steps to reproduce**
1. Visit the URL : https://reactdatepicker.com/
2. Open any narrator (e.g. NVDA or Windows Narrator)
3. Navigate through dates in date picker using arrow keys (Up / Down / Left / Right) on key board to select a date and observe the narration provided by narrator.

**Comment by Cicko** [comment]
Same with JAWS

**Comment by ntanguay** [comment]
Also an issue with Mac VoiceOver

**Comment by ntanguay** [comment]
A workaround for this is to pass in the datepicker prop `adjustDateOnChange: true`. This causes the date to update on a keyboard arrow move, which allows the screen reader to announce the date that is highlighted.

**Comment by saidas3524** [comment]
Any update on this?

**Comment by MahogaWeerasinghe** [comment]
Hi, is that issue resolved or not ?
